code/obj/item/device/pda2/base_os.dm

```diff
@@ -109,25 +110,25 @@
 						if ((!isnull(src.master.uplink)) && (src.master.uplink.active))
 							. += "<a href='byond://?src=\ref[src];note_func=lock'>Lock</a><br>"
 						else
-							. += "<a href='byond://?src=\ref[src];input=note'>Edit</a>"
-							. += " | <a href='byond://?src=\ref[src];note_func=new'>New File</a>"
-							. += " | <a href='byond://?src=\ref[src];note_func=save'>Save</a>"
-							. += " | <a href='byond://?src=\ref[src];note_func=switchmenu'>Load</a><br>"
+							. += {"<a href='byond://?src=\ref[src];input=note'>Edit</a>
+							 | <a href='byond://?src=\ref[src];note_func=new'>New File</a>
+							 | <a href='byond://?src=\ref[src];note_func=save'>Save</a>
+							 | <a href='byond://?src=\ref[src];note_func=switchmenu'>Load</a><br>"}
```

Comment:
unsure if the spaces here will be respected

Reply:
Good point, thankfully it does! 
![image](https://user-images.githubusercontent.com/9563541/86542739-22d24480-becd-11ea-822c-d99a9fb1e1f0.png)